contrib/devtools/gen-manpages.sh

```diff
@@ -21,8 +23,8 @@ $ZCASHD --version | sed -n '1!p' >> footer.h2m
 
 for cmd in $ZCASHD $ZCASHCLI $ZCASHTX; do
   cmdname="${cmd##*/}"
-  help2man -N --version-string=${ZECVER[0]} --include=footer.h2m -o ${MANDIR}/${cmdname}.1 ${cmd}
-  sed -i "s/\\\-${ZECVER[1]}//g" ${MANDIR}/${cmdname}.1
+  help2man -N --version-string=$ZECVER --include=footer.h2m -o ${MANDIR}/${cmdname}.1 ${cmd}
+  sed -i "s/\\\-$ZECCOMMIT//g" ${MANDIR}/${cmdname}.1
```

Comment:
Does this mean for release `v1.0.8-1` the man page will refer to `v1.0.8` instead?